src/cli/cli_commissioner.cpp

```diff
@@ -88,6 +124,20 @@ template <> otError Commissioner::Process<Cmd("joiner")>(Arg aArgs[])
     const otExtAddress *addrPtr = nullptr;
     otJoinerDiscerner   discerner;
 
+    /**
+     * @cli commissioner joiner table
+     * @code
+     * commissioner joiner table
+     * | ID                    | PSKd                             | Expiration |
+     * +-----------------------+----------------------------------+------------+
+     * |                     * |                           J01NME |      81015 |
+     * |      d45e64fa83f81cf7 |                           J01NME |     101204 |
+     * | 0x0000000000000abc/12 |                           J01NME |     114360 |
+     * Done
+     * @endcode
+     * @par
+     * Lists all %Joiner entries in table format.
```

Comment:
What does `%Joiner` do with `%` char?  Does this link to some related definitions or command?

Does it links to cli `joiner` command? If that's the case, I would suggest not doing so since the joiner list here is tied to when device acting as a commissioner (and `joiner` command is for when device is acting as a joiner).

Reply:
@abtink  - the % is actually a workaround so that "Joiner" with an uppercase J doesn't create an unwanted and incorrect link in the generated output. I had to do the same thing with "Commissioner."

Reply:
@abtink for reference: https://openthread.io/reference/cli/contribute#preventing_automatic_links